src/App.tsx

```diff
@@ -1,5 +1,7 @@
 import { useState } from 'react';
 
+import networkList from '~components/Tabber/data';
```

Comment:
Má pratica, importar algo como default usando um nome diferente do arquivo. Pra tudo que não for componente vamos usar named exports